code/game/objects/items/policetape.dm

```diff
@@ -173,7 +173,8 @@
 		for(var/atom/A in T) //Check to see if there's anything solid on the tape's turf (it's possible to build on it)
 			if(A.density)
 				return
-		M:forceMove(T)
+		if (T) // no sending mobs into nullspace!
+			M:forceMove(T)
```

Comment:
>`:`

Just type the var

Reply:
Agreed, I'm not either sure why the proc is typed after mobs since there's nothing mob-specific in there and the parent proc takes an /atom/movable as arg, which can use forceMove().